engine/core/src/main/java/org/datacleaner/job/builder/AnalysisJobBuilder.java

```diff
@@ -1205,4 +1219,37 @@ public Boolean eval(InputColumn<?> inputColumn) {
 
         return result;
     }
+
+    public AnalysisJobBuilder getTopLevelJobBuilder(){
```

Comment:
Ok, Root is better then ;)

I don't know, but adding one without the other seems strange :)